client/my-sites/checkout/concierge-session-nudge/index.jsx

```diff
@@ -30,49 +43,242 @@ export class ConciergeSessionNudge extends React.Component {
+		const { selectedSiteId, isLoading, hasProductsList, hasSitePlans } = this.props;
 		const title = 'Checkout ‹ Expert Session';
 
 		return (
 			<Main className="concierge-session-nudge">
 				<PageViewTracker path="/checkout/:site/add-expert-session/:receipt_id" title={ title } />
 				<DocumentHead title={ title } />
 				<QuerySites siteId={ selectedSiteId } />
+				{ ! hasProductsList && <QueryProductsList /> }
+				{ ! hasSitePlans && <QuerySitePlans siteId={ selectedSiteId } /> }
 
-				<CompactCard>{ this.header() }</CompactCard>
-				<CompactCard>{ this.body() }</CompactCard>
-				<CompactCard>{ this.footer() }</CompactCard>
+				{ isLoading ? (
+					this.renderPlaceholders()
+				) : (
+					<>
+						<CompactCard className="concierge-session-nudge__card-header">
+							{ this.header() }
+						</CompactCard>
+						<CompactCard className="concierge-session-nudge__card-body">
+							{ this.body() }
+						</CompactCard>
+						<CompactCard className="concierge-session-nudge__card-footer">
+							{ this.footer() }
+						</CompactCard>
+					</>
+				) }
 			</Main>
 		);
 	}
 
+	renderPlaceholders() {
+		return (
+			<>
+				<CompactCard>
+					<div className="concierge-session-nudge__header">
+						<div className="concierge-session-nudge__placeholders">
+							<div className="concierge-session-nudge__placeholder-row is-placeholder" />
+						</div>
+					</div>
+				</CompactCard>
+				<CompactCard>
+					<div className="concierge-session-nudge__placeholders">
+						<>
+							<div className="concierge-session-nudge__placeholder-row is-placeholder" />
+							<div className="concierge-session-nudge__placeholder-row is-placeholder" />
+							<div className="concierge-session-nudge__placeholder-row is-placeholder" />
+							<div className="concierge-session-nudge__placeholder-row is-placeholder" />
+						</>
+					</div>
+				</CompactCard>
+				<CompactCard>
+					<div className="concierge-session-nudge__footer">
+						<div className="concierge-session-nudge__placeholders">
+							<div className="concierge-session-nudge__placeholder-button-container">
+								<div className="concierge-session-nudge__placeholder-button is-placeholder" />
+								<div className="concierge-session-nudge__placeholder-button is-placeholder" />
+							</div>
+						</div>
+					</div>
+				</CompactCard>
+			</>
+		);
+	}
+
 	header() {
+		const { translate } = this.props;
 		return (
 			<header className="concierge-session-nudge__header">
-				<h2 className="concierge-session-nudge__title">Offer header</h2>
+				<h2 className="concierge-session-nudge__title">
+					{ translate( 'Congratulations, Your site is being upgraded.' ) }
+				</h2>
 			</header>
 		);
 	}
 
 	body() {
-		return <Fragment>Offer content</Fragment>;
+		const { translate, productCost, productDisplayCost, currencyCode } = this.props;
+		// Full cost should be 150% of base cost, rounded to nearest 50
+		const fullCost = Math.round( ( productCost * 1.5 ) / 50 ) * 50;
```

Comment:
Can we just round it to the nearest `1` instead of `50`?
If `productCost` is **99**, `fullCost` should be **149**, and `savings` should **50**.